devops/openshift/queue-consumers-deploy.yml

```diff
@@ -414,3 +429,9 @@ parameters:
     value: esdc-environment-code
   - name: APPLICATION_ARCHIVE_DAYS
     require: true
+  - name: CAS_BASE_URL
+    required: true
+  - name: CAS_CLIENT_ID
+    required: true
+  - name: CAS_CLIENT_SECRET
+    required: true
```

Comment:
Please follow the same pattern from other parameters and use the `value` with the expected value.